apps/frontend/src/constants.ts

```diff
@@ -12,7 +12,7 @@ export const communitiesHumanReadable: { [key: string]: string } = {
   testing: "Testing",
 };
 
-export const ERROR_SUPPORT_CONTACT = "https://t.me/stevenelleman";
+export const ERROR_SUPPORT_CONTACT = "https://t.me/vivboop";
```

Comment:
I'd be happy to keep helping for Devcon. Maybe make a new constant for EthIndia?

Reply:
all good, i'll handle inbound moving forward!